aws/provider.go

```diff
@@ -418,6 +418,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_redshift_subnet_group":                    resourceAwsRedshiftSubnetGroup(),
 			"aws_route53_delegation_set":                   resourceAwsRoute53DelegationSet(),
 			"aws_route53_record":                           resourceAwsRoute53Record(),
+			"aws_route53_vpc_association_authorization":    resourceAwsRoute53VPCAssociationAuthorization(),
```

Comment:
As mentioned elsewhere, this should probably be `aws_route53_zone_association_authorization`. Even better, would probably be `aws_route53_zone_association_accepter` (depending on what it actually do - I'm not a fan of Go! :).

This because we already have:

```
aws_vpc_peering_connection
aws_vpc_peering_connection_accepter
```

So having the r53 resources being named
```
aws_route53_zone_association
aws_route53_zone_association_accepter
```

makes sense. Pretty if nothing else :D

Reply:
`aws_route53_vpc_association_authorization` would match the name of the API call that creates it, though:

https://docs.aws.amazon.com/Route53/latest/APIReference/API_CreateVPCAssociationAuthorization.html